newrelic/common/signature.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2010 New Relic, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from newrelic.packages import six
+
+if six.PY3:
+    from inspect import Signature
+
+    def bind_args(func, args, kwargs, unwrap=False):
```

Comment:
It looks like unwrap arg is still here even though it was removed in the other implementation and the doc string.